data/data/openstack/workers/variables.tf

```diff
@@ -0,0 +1,9 @@
+variable "worker_server_group_names" {
```

Comment:
We're already in the worker module, we don't need to prefix the variables with `worker`